### DISCUSSION TITLE
Products table order by stock

### DISCUSSION THREAD

**Discussion by kevadamson** [craftcms/commerce#2367] [Ideas]
Can you add functionality so that the product table can be ordered by the stock column? It will give my client an indication on number of products sold. The client is disappointed that "number of products sold" stats aren't part of the core commerce system, so I thought this might be a good interim indication for them.

**Comment by nfourtythree** [comment]
Hi @kevadamson 

Commerce 3 introduced a number of new stats widgets. Two of which might be useful to you.

Top products and top purchasables, the former groups all variants sold into a count grouped by the product. The latter will show a count by each individual purchasable (variant).

Here is an example of the top products widget:

![](https://p191.p3.n0.cdn.getcloudapp.com/items/L1ug4R07/Screenshot+2020-02-20+at+11.57.35.png?v=6d329e517c9bcfc9024c0eff9e226e8e)

We will keep this issue as a feature request, although there are some intricacies with it when it comes to products with multiple variants.
